common/src/main/java/com/wynntils/screens/itemsharing/widgets/SavedItemsHelpButton.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.itemsharing.widgets;
+
+import com.google.common.collect.Lists;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.utils.mc.ComponentUtils;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import java.util.List;
+import net.minecraft.client.gui.GuiGraphics;
+import net.minecraft.network.chat.Component;
+
+public class SavedItemsHelpButton extends WynntilsButton {
```

Comment:
This class could extend `SavedItemsButton`?

Reply:
Or the other way around, either way, I see it as almost 1 to 1 code duplication.